packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -694,6 +694,8 @@ export function commitMount(
     case 'img': {
       if ((newProps: any).src) {
         ((domElement: any): HTMLImageElement).src = (newProps: any).src;
```

Comment:
if srcset is present and this branch is hit, does load trigger? I might have assumed that it would ignore src completely when srcset is present

Reply:
So firefox only triggers load when you set `src` even if there is a `srcset`. So I prefer `src` here to trigger the load in FF. Chrome triggers it when either is set so setting `src` if it exists is good enough but it will fall back if only `srcset` is set. Safari never triggers a new load so the existing solution doesn't work and this change won't really do anything about that

Reply:
mind adding this as a comment?